content/projects/git-exercises/_index.md

```diff
@@ -403,4 +403,18 @@ Your repo should be all of the following:
 
 We just covered the basics here. Please make sure you understand this stuff. It's super important. Git might seem like a weird theoretical thing to a lot of you. It might seem completely unrelated to the actual job of writing code. But it's not. Git makes teamwork on dev teams possible. Without it we'd spend more time shouting at each other than writing useful code. So learn it. Be comfortable with it. When we start working in teams later on all will be made clear.
 
-If you are curious now, spend some time googling git branching strategies. We use the feature branching strategy here. We'll cover it in depth later on in the course.
\ No newline at end of file
+If you are curious now, spend some time googling git branching strategies. We use the feature branching strategy here. We'll cover it in depth later on in the course.
+
+## Instructions For Reviewers
+
+- Ensure that there are 3 branches: main, history, and milkshake-flavours
```

Comment:
Add a line to about the network diagram, we generally check that the branches are present and have been merged to main. We also check that main has been merged back into history.